index.bs

```diff
@@ -73,34 +73,129 @@ command/response format of WebDriver, this permits events to stream
 from the user agent to the controlling software, better matching the
 evented nature of the browser DOM.
 
+Infrastructure {#infrastructure}
+==========================
+
+This specification depends on the Infra Standard. [[!INFRA]]
+
+Network protocol messages are defined using CDDL. [[!RFC8610]]
+
 Protocol {#protocol}
 ==============
 
 This section defines the basic concepts of the WebDriver BiDi
 protocol. These terms are distinct from their representation at the
 <a href=#transport>transport</a> layer.
 
-This specification uses Concise Data Definition Language (CDDL)
-[[!RFC8610]] to describe the format of messages transmitted between the
-[=local end=] and [=remote end=].
+The protocol is defined using [[!RFC8610|CDDL]] schema. For the convenience of
+implementors two seperate schema are defined; the <dfn>remote end schema</dfn>
+which defines the format of messages produced on the [=local end=] and consumed
+on the [=remote end=], and the <dfn>local end schema</dfn> which defines the
+format of messages produced on the [=remote end=] and consumed on the [=local
+end=]
+
+## Schema ## {#protocol-schema}
+
+Issue: Should this be an appendix?
+
+This section gives the initial contents of the [=remote end schema=] and
+[=local end schema=]. These are augmented by the schema fragments defined in
+the remainder of the specification.
+
+[=Remote end schema=]
+
+```
```

Comment:
```suggestion
```cddl
```

I suspect my suggestions won't work, but can we then go with `<pre class=cddl>` like we do for Web IDL, so we can give these boxes some styling?

Reply:
It doesn't work unless pygments supports the type. There are a couple of projects to add CDDL support to pygments, but nothing in the base package. If we found one we like we could perhaps get bikeshed to include it.

Reply:
Would you mind `<pre class=cddl>` which is what we do for Web IDL? For it to be syntax highlighted we'd need to patch Bikeshed, but we could at least draw a box around it with just that.